rmw_dds_common/include/rmw_dds_common/graph_cache.hpp

```diff
@@ -94,6 +95,7 @@ class GraphCache
     const rmw_gid_t & writer_gid,
     const std::string & topic_name,
     const std::string & type_name,
+    const rosidl_type_hash_t & type_hash,
```

Comment:
Here and a few times below, we are adding a new member to an existing, public API, meaning that we are breaking API here.  We could consider `rmw_dds_common` here to be an internal API, but likely things like `rmw_gurumdds` depend on these APIs.  It would be better here if we could tick-tock this somehow.  Could we add new methods that take the hash, and then deprecate these signatures?

Reply:
Yeah, can do. "Tick tock Mr. Wick"